CafeRecords/Pages/Index.cshtml.cs

```diff
@@ -24,13 +24,14 @@ public void OnGet()
         {
             List<Owner> cafeOwners = new List<Owner>();
 
-            string CafeJsonString = GetData("https://data.cityofchicago.org/resource/mqmh-p6ud.json");
-            Cafe[] allCafes = Cafe.FromJson(CafeJsonString);
-            ViewData["allCafes"] = allCafes;
+            string cafeJsonString = GetData("https://data.cityofchicago.org/resource/mqmh-p6ud.json");
```

Comment:
fixed camel casing.

Reply:
Yes... as a variable, it should be camelCase, not PascalCase.